ui/components/AppBar.js

```diff
@@ -0,0 +1,23 @@
+import { AppBar } from '@mui/material';
+import { styled } from "@mui/material/styles";
+import { useTheme } from "@mui/system";
+import theme from '../styles/theme'
+
+const CustomAppBarWrapper = styled(AppBar)(({theme})=>({
+    backgroundColor: '#396679', 
```

Comment:
let's define this color with a meaningful name in the `styles/colors.js` file and then use it here.

Reply:
I have added one @sudo-NithishKarthik can you check it once ?